test/e2e/scenario/vulnerability-overflow/00-assert.yaml

```diff
@@ -18,6 +18,9 @@ collectors:
     command: df -h
   - type: events
     namespace: image-scanner-jobs
+  - type: pod
+    namespace: image-scanner-jobs
+    selector: workload.statnett.no/name=vuln-app
```

Comment:
I think we forgot to add these labels to the podTemplate in https://github.com/statnett/image-scanner-operator/pull/80. Maybe a dedicated fix PR to add that?

Reply:
Yup, I think https://github.com/statnett/image-scanner-operator/pull/83 will fix this.